local-testing/logging-pipeline.md

```diff
@@ -0,0 +1,95 @@
+# Running a logging pipeline locally with Docker
+
+You may wish to test a logging pipeline locally to observe how it deals with
+log messages. The following is a walk-through for running Fluent Bit and
+Elasticsearch locally which can serve as an example for testing other plugins locally.
+
+## Docker Compose
+
+Use [Docker Compose](https://docs.docker.com/compose/) with the following `docker-compose.yaml`:
+
+```yaml
+version: "3.7"
+
+services:
+  fluent-bit:
+    image: fluent/fluent-bit
+    command:
```

Comment:
I personally don't find the command line arguments for FB to be very convenient... I think it's worthwhile to add a few sentences about how you could use a config file and mount it in on a volume

Reply:
Or just change the example to that...

Reply:
Good point.